docker/docker-compose-local.yml

```diff
@@ -1,45 +1,138 @@
 # https://docs.docker.com/compose/compose-file/
 
+x-service-healthy: &service_healthy
+  condition: service_healthy
+
 services:
+  app:
```

Comment:
I keep a single app service instead of splitting it into separate containers (Sidekiq, Webpack, AnyCable, etc.), as that would create many containers for little benefit and make the Docker Compose setup harder to maintain.

Logs remain easy to work with since each process has a distinct prefix (`web.1`, `webpack-ssr.1`, `sidekiq.1`, …) and can be filtered directly:

```bash
docker logs -f web-app-1 | grep sidekiq.1
docker logs -f web-app-1 | grep web.1
docker logs -f web-app-1 | grep webpack-ssr.1
```

<img width="1677" height="554" alt="Image" src="https://github.com/user-attachments/assets/96838c47-c3ee-417c-a882-c27e506d1192" />

<img width="2513" height="256" alt="Image" src="https://github.com/user-attachments/assets/00397e10-1dcf-4871-afc8-d7d8e3c7f2cb" />